src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -429,13 +428,21 @@ public void testSetters() {
         checker.setFileExtensions(".java", "xml");
 
         try {
-            checker.setCharset("UNKNOWN-CHARSET");
+            checker.setCharset("invalid_charset_name");
```

Comment:
Why this change is required?

Reply:
In fact, for the purposes of the issue, it is not required... "UNKNOWN-CHARSET" could also work fine.